src/System.Linq.Expressions/src/System/Runtime/CompilerServices/CallSite.cs

```diff
@@ -415,18 +412,18 @@ private T CreateCustomUpdateDelegate(MethodInfo invoke)
             Expression invokeRule;
 
             Expression getMatch = Expression.Call(
-                typeof(CallSiteOps).GetMethod("GetMatch"),
+                typeof(CallSiteOps).GetMethod(nameof(CallSiteOps.GetMatch)),
```

Comment:
NB: There are opportunities here to cache `MethodInfo` instances. However, when `FEATURE_COMPILE` is enabled, this code runs infrequently (for all `Func` and `Action` delegate types, there's a fast path), so it's not a hot path.